### PR TITLE
refactored burger menu

### PR DESCRIPTION
**Pull request by samueldominguez**
## Description
Refactored burger menu styling and code.

## Changes
- Reduced spacing between menu items
- Replaced hardcoded paths by route enum values

## Checks

- [x] Project Builds
- [x] Project passes tests and checks
- [x] Updated documentation accordingly

### CODE REVIEW

File: src/components/molecules/BurgerMenu.tsx [hunk 1]
```diff
@@ -36,6 +37,8 @@ const MenuItem = ({icon, title, onClick}: MenuItemProps) => {
 const BurgerMenu = ({visible, onHide}: BurgerMenuProps): JSX.Element => {
     const navigate = useNavigate()
     const {t: translate} = useTranslation(namespaces.components.burgerMenu);
+    const menuTitle = 'BALPA'
```
Comment (melinahernandeztheam) [inline src/components/molecules/BurgerMenu.tsx:40]: I would add the menu title to the i18n translations, even if we don't need to translate the title, I think is better to have all the texts in one place, what do you think?

**Review by melinahernandeztheam — DISMISSED** [review]
looks good, just take a look at the comment, is ok if you don't agree, I just want to know your opinion

**Comment by samueldominguez** [comment]
@melinahernandeztheam committed changes. Can you review again?

**Review by aufgfua — APPROVED** [review]
LGTM

**Review by aufgfua — APPROVED** [review]
LGTM (I just fixed some things, and Logout is only redirecting to /login so I created [this issue](https://github.com/TheTributeCommunity/fesbal-frontend/issues/137) so we don't lose track of it)

**Pull request merged**